react-components/select/step4.md

```diff
@@ -0,0 +1,71 @@
+Another customization option, available for all variants, is custom display and data for the Select options.
+
+Let's begin with customizing the data. To change what data is passed around, and back to you on selection, you must implement a SelectOptionObject class. The only required property is a toString function, which converts whatever other data you include in the structure to a string format. This allows the component to internally compare options for filtering, and gives some limited customization for the option's display.
+
+1. Add SelectOptionObject to the list of imports from @patternfly/react-core.
+
+2. Copy the following CustomDataState class, which extends SelectOptionObject, into the App.js file, outside of the App class:
+
+<pre class="file" data-target="clipboard">
+  class CustomDataState implements SelectOptionObject {
+    name: string;
+    abbreviation: string;
+    capital: string;
+    founded: number;
+    constructor(
+      name: string,
+      abbreviation: string,
+      capital: string,
+      founded: number
+    ) {
+      this.name = name;
+      this.abbreviation = abbreviation;
+      this.capital = capital;
+      this.founded = founded;
+    }
+    toString = () =>
+      `${this.name} (${this.abbreviation}) - Founded: ${this.founded}`;
+  }
+</pre>
+
+3. Replace the options state with the following code:
+
+<pre class="file" data-target="clipboard">
```

Comment:
Remind the user which `options state` you're referring to and where it is

Reply:
please clarify..I do not understand the position of the statement.